bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/PWAProxyServlet.java

```diff
@@ -0,0 +1,295 @@
+package com.adobe.acs.commons.cloudservices.pwa.impl;
+
+import com.adobe.granite.confmgr.Conf;
+import com.adobe.granite.confmgr.ConfMgr;
+import com.day.cq.wcm.api.Page;
+import com.day.cq.wcm.api.PageManager;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestPathInfo;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ResourceResolverFactory;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.OptingServlet;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.apache.sling.commons.json.JSONArray;
+import org.apache.sling.commons.json.JSONException;
+import org.apache.sling.commons.json.JSONObject;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.Map;
+/*
+@Component(
+        immediate = true,
+        service = Servlet.class,
+        property = {
+                "sling.servlet.extensions=load",
+                "sling.servlet.extensions=js",
+                "sling.servlet.selectors=pwa",
+                "sling.servlet.selectors=manifest",
+                "sling.servlet.selectors=service-worker",
+                "sling.servlet.methods=" + HttpConstants.METHOD_GET,
+                "sling.servlet.resourceTypes=cq:Page"
+        }
+)*/
+public class PWAProxyServlet extends SlingSafeMethodsServlet implements OptingServlet {
```

Comment:
I think we can delete this file now? It should've been moved out to other servlets (separation of concerns)